modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -213,41 +216,44 @@ compression parameters :
         return 0;
     }
 @endcode
- */
+@param filename Name of the file.
+@param img Image to be saved.
+@param params Format-specific save parameters encoded as pairs, see cv::ImwriteFlags
+paramId_1, paramValue_1, paramId_2, paramValue_2, ... .
+*/
 CV_EXPORTS_W bool imwrite( const String& filename, InputArray img,
               const std::vector<int>& params = std::vector<int>());
 
-/** @overload */
-CV_EXPORTS_W Mat imdecode( InputArray buf, int flags );
-
 /** @brief Reads an image from a buffer in memory.
 
-@param buf Input array or vector of bytes.
-@param flags The same flags as in imread, see @ref cv::ImreadModes.
-@param dst The optional output placeholder for the decoded matrix. It can save the image
-reallocations when the function is called repeatedly for images of the same size.
-
-The function reads an image from the specified buffer in the memory. If the buffer is too short or
+The function imdecode reads an image from the specified buffer in the memory. If the buffer is too short or
 contains invalid data, the empty matrix/image is returned.
 
-See imread for the list of supported formats and flags description.
+See cv::imread for the list of supported formats and flags description.
 
-@note In the case of color images, the decoded images will have the channels stored in B G R order.
+@note In the case of color images, the decoded images will have the channels stored in **B G R** order.
+@param buf Input array or vector of bytes.
+@param flags The same flags as in cv::imread, see cv::ImreadModes.
  */
+CV_EXPORTS_W Mat imdecode( InputArray buf, int flags );
+
+/** @overload
+@param buf
+@param flags
+@param dst The optional output placeholder for the decoded matrix. It can save the image
+reallocations when the function is called repeatedly for images of the same size.
+*/
 CV_EXPORTS Mat imdecode( InputArray buf, int flags, Mat* dst);
 
 /** @brief Encodes an image into a memory buffer.
 
+The function imencode compresses the image and stores it in the memory buffer that is resized to fit the
+result. See cv::imwrite for the list of supported formats and flags description.
+
 @param ext File extension that defines the output format.
 @param img Image to be written.
 @param buf Output buffer resized to fit the compressed image.
-@param params Format-specific parameters. See imwrite and @ref cv::ImwriteFlags.
-
-The function compresses the image and stores it in the memory buffer that is resized to fit the
-result. See imwrite for the list of supported formats and flags description.
-
-@note cvEncodeImage returns single-row matrix of type CV_8UC1 that contains encoded image as array
-of bytes.
+@param params Format-specific parameters. See imwrite and cv::ImwriteFlags.
  */
```

Comment:
is there a reson for space here " */" ? which is better "*/" or " */". generally like " */" i wondered why ?